src/shell/components/global-tabs/GlobalTabs.tsx

```diff
@@ -103,6 +104,7 @@ export default memo(function GlobalTabs() {
 
   return (
     <ThemeProvider theme={theme}>
+      <GlobalDirtyCodeModal />
```

Comment:
These should probably be in the code app so it is co-located towards the code it operates on.

Reply:
This is the modal that activates when you try to unpin an dirty code tab. It is NOT the modal that activates when you navigate away from a dirty code file. It is necessary for this modal to be available anywhere in the app where a user could possibly unpin a tab. For that reason it cannot be in code; because it would be unmounted and then unavailable outside of Code. 

According to our convo earlier today, you don’t believe this modal should exist. I will wait for direction on that based on Zosh’s user feedback sessions this week.